examples/item/item.json

```diff
@@ -186,7 +197,7 @@
             0.0,
             358.59375
           ],
-          "step": 1.40625,
+          "step": 1.4007568359375,
```

Comment:
This should probably be trunctated/rounded to a reasonable precision for geographic data. If this is degrees, then thirteen digits is -- [very small](https://xkcd.com/2170/) :-).

Reply:
I generally agree, but I think it is better to accurately report on what the underlying netCDF reports, otherwise there would be inconsistencies with the data.

But I guess this can be resolved with the outcome of [this comment](https://github.com/stactools-packages/datacube/pull/8#discussion_r1161658665).

Reply:
Nah, you're right, if it comes straight from the NetCDF then it's fine to keep it overly precise.